libaleth/SendDialog.ui

```diff
@@ -2,6 +2,32 @@
 <ui version="4.0">
  <class>SendDialog</class>
  <widget class="QDialog" name="SendDialog">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>223</width>
```

Comment:
@winsvega SendDialog can't be resized. at least on my version. are you sure you check the latest one? or you meant AlethOn? Do you think we should fix the height for AlethOne?